src/inference.rs

```diff
@@ -236,6 +237,57 @@ impl<B: Backend> FSRS<B> {
     pub fn current_retrievability(&self, state: MemoryState, days_elapsed: u32) -> f32 {
         (days_elapsed as f32 / (state.stability * 9.0) + 1.0).powi(-1)
     }
+
+    /// Compare the current model and weights with another set of weights.
+    /// Returns a tuple with a boolean indicating whether the current model is better than the other,
```

Comment:
I think it would be clearer if the returned bool was true if the new weights are better, and then the docs could be updated:

```suggestion
    /// Returns a tuple with a boolean that's true if the provided weights are better
```

Alternatively, since you're already returning the two universal values, perhaps the boolean is not required, and the docstring could be updated instead, to something like:

> Returns the universal metrics for the existing and provided parameters. If the first value is smaller than the second value, the provided parameters are worse than the previous ones.

Reply:
In my view, the suggested comment is confusing, because I don't know what the provided weights means. Does it mean the input parameters? If so, the comment is wrong because the boolean is true when the input parameters are worse than the parameters of the model itself.

Reply:
> Returns the universal metrics for the existing and provided parameters. If the first value is smaller than the second value, the provided parameters are worse than the previous ones.

I prefer this one. I will update the code.